draft-bft-rats-kat.md

```diff
@@ -145,49 +161,63 @@ related entities.
-Attestation Token (PAT) by the Platform Attestation Service component. The PAT needs
-to be linked to the Key Attestation Token (KAT). The Key Attestation Token (KAT) includes
-the public key of the IK (pkT) and is then signed with the Key Attestation Key (KAK).
-
-To ensure freshness of the PAT and the KAT a nonce is used, as suggested by the RATS
-architecture {{-rats-arch}}. Here is the symbolic API call to request a KAT and a PAT, which
+Attestation Token (PAT) by the Platform Attestation Service.
+The PAT needs to be linked to the Key Attestation Token (KAT)
+and this linkage can occur in a number of ways. One approach
+is described in this specification in {{bundle}}. The Key 
+Attestation Token (KAT) includes the public key of the IK 
+(pkT) and is then signed with the Key Attestation Key (KAK).
+
+To ensure freshness of the PAT and the KAT a nonce is used,
+as suggested by the RATS architecture {{-rats-arch}}. Here
+is the symbolic API call to request a KAT and a PAT, which
 are concatinated together as the CAB.
 
 ~~~~
 cab = createCAB(key_id, nonce)
 ~~~~
 
-Once the CAB has been sent by the Presenter to the Recipient, the Presenter has to
-demonstrate possession of the private key. The signature operation uses the private
-key of the IK (skT). How this proof-of-possession of the private key is accomplished
-depends on the details of the usage protocol and is outside the scope of this specification.
-
-The Recipient of the CAB and the proof-of-possession data (such as a digital signature)
-first extracts the PAT and the KAT. The PAT and the KAT may need to be
-conveyed to a Verifier. If the PAT is in the form of attestation results the checks can
-be performed locally at the Recipient, whereby the following checks are made:
-
-- The signature protecting the PAT MUST pass verification when using available trust anchor(s).
-- The PAT MUST be checked for replays, which can be checked by comparing the nonce included
-  in one of the claims and matching it against the nonce provided to the attester.
-- The claims in the PAT MUST be matched against stored reference values.
+Once the CAB has been sent by the Presenter to the Recipient,
+the Presenter has to demonstrate possession of the private key.
+The signature operation uses the private key of the IK (skT).
+How this proof-of-possession of the private key is accomplished
+depends on the details of the usage protocol and is outside the
+scope of this specification.
+
+The Recipient of the CAB and the proof-of-possession data (such
+as a digital signature) first extracts the PAT and the KAT. The
+PAT and the KAT may need to be conveyed to a Verifier. If the 
+PAT is in the form of attestation results the checks can be 
+performed locally at the Recipient, whereby the following checks
+are made:
+
+- The signature protecting the PAT MUST pass verification when 
+using available trust anchor(s).
+- The PAT MUST be checked for replays, which can be checked by 
```

Comment:
IIUC, the bundling method we describe reuses the PAT across multiple KATs, so the session freshness for the "usage protocol" is to be checked in the KAT only.

This step should be "PAT and KAT chaining is checked"